src/pages/kb/open-source/admin-guide/secrets.md

```diff
@@ -0,0 +1,63 @@
+---
+category: admin-guide
+parent_category: open-source
+title: Secrets Keys in Redash
+slug: secrets
+toc: true
+---
+
+## Background
+Redash encrypts secret information with two keys: the Cookie Secret and the Data Source Secret.
+
+### Cookie Secret 
+
+The cookie secret is taken from the `REDASH_COOKIE_SECRET` environment variable. It is used for various cryptographic features of the web server, such as authenticating users, signing cookies, and storing user session information. It is required to start the application.
+
+If you attempt to start Redash without it, the webserver will not initialise and you will find the following message in your logs:
+
+```bash
+Exception: You must set the REDASH_COOKIE_SECRET environment variable. Visit http://redash.io/help/open-source/admin-guide/secrets for more information.
+```
+
+To fix this message: create an environment variable called `REDASH_COOKIE_SECRET` and give it a value.
+### Data Source Secret
+
+The data source secret is taken from the `REDASH_SECRET_KEY` environment variable. It is used to encrypt database passwords, API keys, connection strings, and any other secret fields found on the Settings > Data Sources screen.
+
+It is required to start the application. However, if you do not set one explicitly, Redash will use the cookie secret instead. This is helpful for development. But for maximum security, we recommend you set a unique value for both variables.
+
+
+## Choosing a Secret Key
+
+Treat these secret keys like passwords. A strong secret key will be unique and hard to guess. Do not reuse secret keys across instances of Redash or commit them to version control. If a hostile actor guesses your secret key they might compromise your Redash instance. 
+
+Our cloud images use the CLI tool `pwgen` to generate secret keys, but any [strong password generator](https://duckduckgo.com/?q=pwgen+32+strong) will work:
```

Comment:
Is this aligned w/ Flask's recommendations?

Reply:
[Here's](https://flask.palletsprojects.com/en/2.0.x/tutorial/deploy/#configure-the-secret-key) what they recommend. I'm updating our guidance to mirror it.